x-pack/platform/packages/shared/kbn-streams-schema/src/models/ingest/api.ts

```diff
@@ -89,15 +94,16 @@ interface UnwiredStreamGetResponse extends StreamGetResponseBase {
 type IngestStreamGetResponse = WiredStreamGetResponse | UnwiredStreamGetResponse;
 
 /**
- * Ingest stream upsert request
+ * PUT /streams/{name}
+ * Wired or Unwired (Ingest) Stream Upsert Request
+ * Don't confuse that with IngestUpsertRequest
  */
-
 interface UnwiredStreamUpsertRequest extends StreamUpsertRequestBase {
-  stream: UnwiredIngestUpsertRequest;
+  stream: UnwiredIngestUpsertRequest & { description: string };
```

Comment:
💬  `UnwiredIngestUpsertRequest` is the type used by the /_ingest route as well, therefore `description` cannot be specified there. 
How would you name this type `UnwiredIngestUpsertRequest & { description: string };` ?

Reply:
> UnwiredIngestUpsertRequest is the type used by the /_ingest route as well, therefore description cannot be specified there.

UnwiredIngestUpsertRequestStream? (yuck but also...)

Reply:
updated in https://github.com/elastic/kibana/pull/215240/commits/95d187d79c6ffc1bbdb4a4c7d90729d1d6c8ef6e